### PR TITLE
Added SteamAgeRevolution Docs

### PR DESCRIPTION
**Pull request by MasterEnderman**
Proper Documentation for the current SteamAgeRevolution Machines.

### CODE REVIEW

**Comment by MasterEnderman** [comment]
Done.

File: docs/Mods/SteamAgeRevolution/AlloyForge.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+The Alloy Forge is a machine that converts two input fluids into one output fluid.
+
+## Calling
+
+You can call the Alloy Forge package using `mods.steamagerevolution.AlloyForge`.
+
+## Removing
+
+This function removes the first recipe they find with the given [ILiquidStack](/Vanilla/Items/ILiquidStack/) `output`:
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/AlloyForge.md]:
```suggestion
This function removes the first recipe they find with the given [ILiquidStack](/Vanilla/Liquids/ILiquidStack/) `output`:
```

File: docs/Mods/SteamAgeRevolution/Crucible.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+The Crucible is a machine that converts an input item into an output fluid using steam.
+
+## Calling
+
+You can call the Crucible package using `mods.steamagerevolution.Crucible`.
+
+## Removing
+
+This function removes the first recipe they find with the given [ILiquidStack](/Vanilla/Items/ILiquidStack/) `output`:
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/Crucible.md]:
```suggestion
This function removes the first recipe they find with the given [ILiquidStack](/Vanilla/Liquids/ILiquidStack/) `output`:
```

File: docs/Mods/SteamAgeRevolution/Distiller.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+The Distiller is a machine that converts an input fluid into an output fluid and item using steam.
+
+## Calling
+
+You can call the Distiller package using `mods.steamagerevolution.Distiller`.
+
+## Removing
+
+This function removes the first recipe they find with the given [IItemStack](/Vanilla/Items/IItemStack/) `outputStack` and [ILiquidStack](/Vanilla/Items/ILiquidStack/) `output`:
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/Distiller.md]:
```suggestion
This function removes the first recipe they find with the given [IItemStack](/Vanilla/Items/IItemStack/) `outputStack` and [ILiquidStack](/Vanilla/Liquids/ILiquidStack/) `output`:
```

File: docs/Mods/SteamAgeRevolution/Vat.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+The Vat is a machine that converts input items and fluids into an output fluid.
+
+## Calling
+
+You can call the Vat package using `mods.steamagerevolution.Vat`.
+
+## Removing
+
+This function removes the first recipe they find with the given [ILiquidStack](/Vanilla/Items/ILiquidStack/) `output`:
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/Vat.md]:
```suggestion
This function removes the first recipe they find with the given [ILiquidStack](/Vanilla/Liquids/ILiquidStack/) `output`:
```

**Pull request closed without merging**